src/hooks/updates/useEvaluateResourceConfigUpdates.ts

```diff
@@ -0,0 +1,60 @@
+import { useEntityType } from 'context/EntityContext';
+import useGlobalSearchParams, {
+    GlobalSearchParams,
+} from 'hooks/searchParams/useGlobalSearchParams';
+import useDraftSpecs from 'hooks/useDraftSpecs';
+import { useLiveSpecsExtWithSpec } from 'hooks/useLiveSpecsExt';
+import { isEqual } from 'lodash';
+import { useMemo } from 'react';
+import { ResourceConfigDictionary } from 'stores/ResourceConfig';
+
+const evaluateResourceConfigEquality = (
```

Comment:
This is fine for now - but eventually this file should be moved. I think it is pretty weird to have a subfolder under `hooks` called `updated`. That would read to me as being hooks for running updates.

I also think if we're going do a path (which I think we should) where sections have all these hooks, stores, etc. we should make the section/entity the parent folder.